cluster/gce/util.sh

```diff
@@ -912,6 +912,19 @@ function detect-subnetworks() {
   echo "${color_red}Could not find subnetwork with region ${REGION}, network ${NETWORK}, and project ${NETWORK_PROJECT}"
 }
 
+function delete-all-firewall-rules-in-network() {
```

Comment:
Maybe name this `delete-all-firewall-rules`? `in-network` seems unnecessary also given that it's not actually taking the network as an argument.

Reply:
Sounds good, done.